docs/source/performance.md

```diff
@@ -0,0 +1,291 @@
+# Performance considerations
+
+There are various aspects of a Hyperledger Fabric network that affect the performance of the overall system/ Performance in a Hyperledger Fabric network is complex because in an appropriately deployed network there will be many organizations participating, each with their own hardware and networking infrastructure, along with different solution characteristics such as number of channels, chaincode implementations and policies.
+
+This section will cover some of the considerations that need to be made. It doesn't cover some scalability aspects such as impacts of large numbers of Private Data Collections, number of organizations in a network for example.
+
+## Hardware considerations
+
+Each organization may provide it's own hardware to host their peer nodes and ordering service nodes. Ordering service nodes may be provided by a single organization, or may be contributed from various organizations. It's important to realise that individual organizations could impact the whole network (depending on policies such as endorsement policies), therefore each organization must ensure that they provide adequate resources for the services they deploy.
+
+### Persistent storage
+
+As Hyperledger Fabric performs a lot of disk I/O it goes without saying that you should ensure you are using the fastest disk storage available to you and that the physical drive it ultimately resides on should be SSD. If you are using network attached storage then ensure that you choose the fastest IOPs available.
```

Comment:
> and that the physical drive it ultimately resides on should be SSD

these specific recommendations age very quickly - SSD is fast today and slow tomorrow compared to a new storage technology. Let's just limit to the bit about being fast (and you can explain in terms of what - latency/throughput etc)

Reply:
Thanks @ale-linux I'll remove the mention of SSD